.github/workflows/stale.yml

```diff
@@ -47,3 +47,4 @@ jobs:
           close-issue-message: >
             This issue has been closed because it has not received any activity in the last 14 days
             since being marked as 'stale'
+          ascending: true
```

Comment:
It does make sense to close the older stale issues first. But I was wondering was there any default limit to number of issues to be processed? Why it is not closing all the issues that are older than 180 days and stale for 14 days.

Reply:
The default value is 30 as per https://github.com/actions/stale#operations-per-run

Reply:
the workflow triggers daily at midnight, considering 30 issues a day. It should have closed  by now?

Reply:
With the default descending order, it always checks the latest issues and only a small amount can be closed. Older issues are never checked.